docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1319,6 +1319,18 @@ SELECT *
 FROM example.testdb.customer_orders FOR VERSION AS OF 8954597067493422955
 ```
 
+Iceberg supports named references of snapshots via branches and tags.
+Time travel can be performed to branches and tags in the table. 
+Note that when time traveling to a branch, the snapshot which is resolved is the head of the branch.
```

Comment:
I'm a little confused by this last sentence. Could you elaborate on that?

Reply:
https://github.com/apache/iceberg/pull/5364#issuecomment-1227902420

@amogh-jahagirdar, let's try to do a slightly condensed version of this comment to explain how our behavior is driven by a retention policy and is not a direct parallel to git branching behavior.

Later, I want to take this comment and elaborate with some visuals to explain this behavior in more detail, and we can shorten the explanation and link to the Iceberg docs at that time.

Reply:
@amogh-jahagirdar, I spoke with @ebyhr on the side and he's just gonna remove that line for now and you and I should follow up and make an easy reference point in the Iceberg docs to point to about this.

https://github.com/apache/iceberg/issues/8638